someip2val/proto/kuksa_data_broker/kuksa/val/v1/types.proto

```diff
@@ -0,0 +1,286 @@
+/********************************************************************************
```

Comment:
From a long term perspective I do not see it as optimal that we copy proto files so they exist in multiple repositories (but ok for now). But what would be a better approach? Having the proto-files in separate repository so that both KUKSA.val and kuksa.val.feeders can include it as a submodule, or by some other fetch-mechanism? Or a setup script that feteches the right version from kuksa.val. Any ideas @SebastianSchildt?

Reply:
protos are all over the place, some with symlinks, I may try fetching them with CMake, but not in this PR